samples/idtokens-serverless.js

```diff
@@ -29,14 +29,15 @@ function main(
    * TODO(developer): Uncomment these variables before running the sample.
    */
   // const url = 'https://TARGET_URL';
+  // const targetAudience = null;
```

Comment:
As written, this code would fail because we're using a `const` instead of `let`, and trying to assign down on line 40.  Would another option here be moving `const {URL} = require('url');` with the other requires, and always doing the assignment of `const targetAudience = new URL(url)` on line 32?